openapi-v0.yaml

```diff
@@ -47,80 +47,105 @@ components:
+          description: Name of the newsletter
         cancelled:
           type: boolean
           example: false
+          description: Indicates if the newsletter has been cancelled. This prevents it from being actively promoted across the site and prevents sign-ups and unsubscribes.
         restricted:
           type: boolean
           example: false
+          description: Indicates if the newsletter is restricted (e.g. not available to the public, requiring payment/subscription, or requiring a registered account).
         paused:
           type: boolean
           example: false
+          description: Indicates if the newsletter is paused (temporarily not running). This prevents it from being actively promoted across the site and prevents most sign-ups.
         emailConfirmation:
           type: boolean
           example: false
+          description: Indicates if the newsletter sign up process requires an email-confirmation step in order to subscribe.
         brazeNewsletterName:
           type: string
           example: Editorial_MorningBriefingUK
+          description: The newsletter name identifier used to create segments in Braze
         brazeSubscribeAttributeName:
           type: string
           example: MorningBriefingUk_Subscribe_Email
+          description: Used for custom attributes in Braze. Detemines whether a user is a subscriber to a newsletter or not.
         brazeSubscribeEventNamePrefix:
           type: string
           example: morning_briefing_uk
+          description: Used for custom events in Braze. Detemines whether a user is a subscriber to a newsletter or not.
         theme:
           type: string
           example: News
+          description: Describes the newsletter theme (similar to a pillar) which is a top level category.
         group:
           type: string
           example: News in brief
+          description: Describes the newsletter group. Used in the all newsletters page as a secondary level grouping to the theme.
         description:
           type: string
           example: Archie Bland and Nimo Omer take you through the top stories and what they mean, free every weekday morning
+          description: Provides a short description of the newsletter for promotion across the site.
         regionFocus:
           type: string
           example: UK
+          description: Indicates whether the newsletter content is tailored towards a particular region.
         frequency:
           type: string
           example: Every weekday
+          description: Indicates how often the newsletter is sent out.
         listIdV1:
           type: integer
           example: 3640
+          description: A legacy, numerical unique identifier. Is not widely used but some usage still exists.
+          deprecated: true
         listId:
           type: integer
           example: 4156
+          description: A numerical unique identifier for a newsletter.
```

Comment:
Is it worth mentioning that this is used to identify the signup embed (on thrashers)?